src/app.rs

```diff
@@ -25,9 +23,15 @@ pub enum AppModes{
 }
 
 pub fn get_frame_lines(
-            ref_transform: &Arc<RwLock<rosrust_msg::geometry_msgs::Transform>>, axis_length: f64
+            tf_listener: Arc<rustros_tf::TfListener>,
+            static_frame: String,
+            axis_length: f64
         ) -> Vec<Line> {
-        let tf = &ref_transform.as_ref().read().unwrap();
+        let tf = tf_listener.lookup_transform(
+            &static_frame,
+            "base_link",
+            rosrust::Time::new()
```

Comment:
is this. ros::Time(0) in rosrust?

Reply:
Time { sec: 0, nsec: 0 }
I would say so